app/models/registration_wizard.rb

```diff
@@ -95,31 +95,29 @@ def answers
                             value: store["confirmed_email"],
                             change_step: :contact_details)
 
-    unless query_store.works_in_school?
-      if inside_catchment? && query_store.works_in_childcare?
-        array << OpenStruct.new(key: "Do you work in a nursery?",
-                                value: store["works_in_nursery"].capitalize,
-                                change_step: :work_in_nursery)
-
-        if query_store.works_in_nursery?
-          kind_of_nursery = store["kind_of_nursery"]
-
-          array << OpenStruct.new(key: "Type of nursery",
-                                  value: I18n.t("registration_wizard.kind_of_nursery.#{kind_of_nursery}"),
-                                  change_step: :kind_of_nursery)
-        end
-
-        if query_store.kind_of_nursery_private? || !query_store.works_in_nursery?
-          array << if query_store.has_ofsted_urn?
-                     OpenStruct.new(key: "Ofsted registration details",
-                                    value: institution_from_store.registration_details,
-                                    change_step: :have_ofsted_urn)
-                   else
-                     OpenStruct.new(key: "Do you have a URN?",
-                                    value: store["has_ofsted_urn"].capitalize,
-                                    change_step: :have_ofsted_urn)
-                   end
-        end
+    if inside_catchment? && query_store.works_in_childcare? && !query_store.works_in_school?
```

Comment:
Since works_in_school and works_in_childcare are exclusive now, can !query_store.works_in_school? be removed?

Reply:
Yep, good call.